performance/reports/rp.py

```diff
@@ -23,7 +24,7 @@ def is_loa2(rp):
     return rp not in LOA1_RP_LIST
 
 
-def get_rps_with_successes(df_verifications_by_rp):
+def get_rp_names_from_df(df_verifications_by_rp):
```

Comment:
(Existing issue rather than new in this PR:) seems to me that we keep calling this function, and really we ought to call it once, and pass around the result. Or store it somewhere, or something.

Reply:
This is a general purpose function (renamed to make it sound so), and is being invoked to work on different DataFrames, hence we can't really cache the result.